glide.yaml

```diff
@@ -0,0 +1,7 @@
+package: github.com/wavefronthq/wavefront-opentracing-sdk-go
```

Comment:
Can you use dep instead? https://github.com/golang/dep

Reply:
Dep doesn't work with opentrace package... don't know why...

Reply:
solved